brainstormforce/astra
# AST-3159 - Fix Author Link not working on layout 2

### Description
Test Data: (If any)  

Clicking the Author link leads to a not found page, instead of the Author page.

Steps To Reproduce: 

1. Go to Post title Customizer
2. And Just switch the Layout 2
3. Check the author link by clicking on Author name.

### Screenshots
https://share.getcloudapp.com/ApuvRobK

### Types of changes
On layout 2 , the function is not working out astra_post_author. Made some changes use get_transient and set_transient to optimized futher.

### How has this been tested?

> Checked with Post Title Layout 1
> Also with Post Title Layout 2

### Checklist:
